src/main/java/hellos/Hellos.java

```diff
@@ -42,6 +48,9 @@ private String nicSaysHello() {
   private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
+  private String DanteSaysHello() {
```

Comment:
I _would_ put a blank link between method definitions, though.

```suggestion

  private String DanteSaysHello() {
```